scripts/prepare_automatic_backups.py

```diff
@@ -44,6 +52,12 @@ def generate_backup_url(cloud_storage_bucket_name, module_class_names):
 
 
 def update_cron_dict(cron_dict):
+    """Update yaml file content by updating argument passed content which is having yaml 
+    file content. 
+    
+    Args:
+      cron_dict : type=dictionary. content of yaml file in dictionary type. 
```

Comment:
Ditto. Please make correction in the format.

Reply:
resolved it.